infrastructure-playbooks/purge-docker-cluster.yml

```diff
@@ -360,29 +360,21 @@
       enabled: no
     when: not is_atomic
 
-  - name: remove docker-py
+  - name: remove docker-py on Debian
     pip:
       name: docker-py
-      version: 1.1.0
       state: absent
     when:
-      ansible_version['full'] | version_compare('2.1.0.0', '<') and
-      not is_atomic
+      - not is_atomic
```

Comment:
Use `or` instead of `and`? or maybe just saying `ansible_distribution == 'Debian'` is enough?